packages/regenerator-transform/src/util.js

```diff
@@ -23,10 +25,30 @@ export function getTypes() {
   return currentTypes;
 }
 
-export function runtimeProperty(name) {
+const runtimeNames = new WeakMap();
+
+export function runtimeProperty(name, scope, opts) {
```

Comment:
Now that `runtimeProperty` takes options and does more than just construct a member expression, I think we should move it out of `util.js` and make it a method of the `Emitter` class. That way we won't need to pass the options explicitly, since they're stored as a property of the `Emitter` object now. From what I can tell, everywhere `runtimeProperty` was previously called, we have access to an `Emitter` instance, so this change should be pretty straightforward. What do you think @nicolo-ribaudo?